### ISSUE TITLE
the first pass model performed better than the second pass

### ISSUE THREAD

**Issue by Kikyo-16** [DCASE-REPO/dcase2018_baseline#16]
Hello，I found different results from the paper（Large-Scale Weakly Labeled Semi-Supervised Sound
Event Detection in Domestic Environments）. 
As mentioned in the article, the F1-score performance of the baseline system after the first pass is 7.51, I got 18.6，which is higher than the second pass. I am very confused, I just stacked the result of the first model on each frame.Is it a scoring loophole？

**Comment by turpaultn** [comment]
Hello, 
Are you using GPU to train the model ?
We saw a lot of variation in the results (first or second pass) when using GPU, so in that case it is possible.

**Comment by Kikyo-16** [comment]
Yes. But in the same experiment, the second pass just got 12.0. It does this mean that the second model has no effect on improvement？

**Comment by turpaultn** [comment]
For the event-based F1-score, it is possible. 
This model uses a naive idea by tagging the unlabeled data. You can check the tagging result of your first pass  model for example which can give you clues about how the second pass can perform or not (i.e if the first pass model overfit or give a lot of wrong labels, it is definitely possible that the second pass will have worse results).

I would advise to check tagging results first, because the models are trained for that. The trick of taking the frame level output and smoothing it a bit is also a basic idea, so you could have a really good audio tagging model which gives us really bad boundaries.

These are just some ideas, we can discuss it more in detail if you want, but this is not a scoring loophole, it is coming from the way the model has been designed (it is a baseline, we can point out many drawbacks).

**Comment by Kikyo-16** [comment]
Thanks, I got it~

**Comment by Kikyo-16** [comment]
Thanks, I got it~
